src/snowflake/cli/plugins/spcs/image_repository/commands.py

```diff
@@ -0,0 +1,128 @@
+import json
+from typing import Optional
+
+import requests
+import typer
+from click import ClickException
+from snowflake.cli.api.commands.decorators import (
+    global_options_with_connection,
+    with_output,
+)
+from snowflake.cli.api.commands.flags import DEFAULT_CONTEXT_SETTINGS
+from snowflake.cli.api.output.types import CollectionResult
+from snowflake.cli.plugins.spcs.image_registry.manager import RegistryManager
+from snowflake.cli.plugins.spcs.image_repository.manager import ImageRepositoryManager
+
+app = typer.Typer(
+    context_settings=DEFAULT_CONTEXT_SETTINGS,
+    name="image-repository",
+    help="Manages Snowpark Container Services image repositories.",
+    rich_markup_mode="markdown",
+)
+
+
+@app.command("list-images")
+@with_output
+@global_options_with_connection
+def list_images(
+    repo_name: str = typer.Option(
+        ...,
+        "--repository_name",
```

Comment:
As this command is moved to image_repository folder. I think we should make name an argument instead of an option.
`snow spec image-repository list-images <repo_name>`